model/lifecycle.go

```diff
@@ -653,8 +653,13 @@ func CreateBuildFromVersionNoInsert(args BuildCreateArgs) (*build.Build, task.Ta
 	}
 	b.BuildNumber = strconv.FormatUint(buildNumber, 10)
 
+	// no tasks available for build
+	if len(args.Aliases) > 0 && len(args.TaskNames) == 0 {
```

Comment:
would it be better to check this earlier on in https://github.com/evergreen-ci/evergreen/blob/c8d081a3e7ab6fe37d1b952e1e626f6cc76253db/model/patch_lifecycle.go#L397? you already do this in the repotracker

Reply:
Good point. I think I'd rather put the check higher up in this function and remove it from the repotracker, just to ensure we have a catch-all (although it looks like only the repotracker populates Aliases, but that may change for whatever reason so)